todo_app/templates/index.html

```diff
@@ -7,11 +7,31 @@ <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
+  <div class="mb-5">
+    <h2 class="font-weight-light">Add To-Do Item</h2>
+    <form action="/" method="post">
+        <input type="text" class="form-control mb-1" name="add-item">
+        <button type="submit" class="btn btn-outline-primary">Submit</button>
+    </form>
+  </div>
+
   <div class="row">
     <div class="col-md-12">
-      <h2>Items</h2>
+      <h2 class="font-weight-light">Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+        {% for todo_item in todo_items %}
+          <li class="list-group mb-3">{{todo_item.id }} - {{todo_item.status}} - {{todo_item.title}}
```

Comment:
Really minor, but from a UX perspective does adding the id here actually help a user?